### PR TITLE
Clean docs up a bit

### PR DESCRIPTION
**Pull request by matthchr**
**If applicable**:
- [x] this PR contains documentation
- [ ] this PR contains tests

### CODE REVIEW

File: docs/hugo/content/guide/aso-controller-settings-options.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+AZURE_SYNC_PERIOD is the frequency at which resources are re-reconciled with Azure when
+there have been no triggering changes in the Kubernetes resources.
+This sync exists to detect and correct changes that happened in Azure that Kubernetes is not aware about.
+BE VERY CAREFUL setting this value low - even a modest number of resources can cause
+subscription level throttling if they are re-synced frequently. If nil or empty (`""`), sync period defaults to `1h`.
+
+**Format:** `duration string`
+
+**Example:** `"1h"`, `"15m"`, or `"60s"`. See https://pkg.go.dev/time#ParseDuration for more details.
```
Comment (theunrepentantgeek) [inline docs/hugo/content/guide/aso-controller-settings-options.md:75]: Shouldn't this be a link?

File: docs/hugo/content/guide/aso-controller-settings-options.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+**Required**: False
+
+### AZURE_OPERATOR_MODE
+
+AZURE_OPERATOR_MODE determines whether the operator should run _watchers_, _webhooks_ or _both_ (default). An empty string, or any unrecognized value, means _both_.
+
+**Format:** `webhooks|watchers|both`
+
+**Example:** `"webhooks"`, `"watchers"` or `"both"`
```
Comment (theunrepentantgeek) [inline docs/hugo/content/guide/aso-controller-settings-options.md:85]:
```suggestion
**Examples:** `"webhooks"`, `"watchers"` or `"both"`
```

File: docs/hugo/content/guide/aso-controller-settings-options.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+**Required**: False
+
+### AZURE_TARGET_NAMESPACES
+
+AZURE_TARGET_NAMESPACES lists the namespaces the operator will watch for Azure resources (if the mode includes running watchers). 
+If it's empty the operator will watch all namespaces.
+
+**Format:** `string` (comma-separated namespace names)
```
Comment (theunrepentantgeek) [inline docs/hugo/content/guide/aso-controller-settings-options.md:94]: Do we tolerate trailing spaces after the `,` ?

If we do, mention that; if we don't, let's log an issue so we do in the next release.

File: docs/hugo/content/guide/frequently-asked-questions.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 ---
 title: FAQ
+weight: 0
+menu:
+  main:
+    weight: 0
```
Comment (theunrepentantgeek) [inline docs/hugo/content/guide/frequently-asked-questions.md:6]: Be explicit about the weight here to ensure the ordering in the menu is the one we want. 
I think I've numbered existing top level items in increments of `10`.

File: docs/hugo/content/guide/installing-from-helm.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Installation: From Helm"
+weight: -2
+menu:
+  main:
+    weight: -2
```
Comment (theunrepentantgeek) [inline docs/hugo/content/guide/installing-from-helm.md:6]: I think we're getting too many things in the header menu here, we need to keep it down to around five:

image

**Review by theunrepentantgeek — CHANGES_REQUESTED** [review]
The main menu is shown in the header of every page, it's not big enough to have 11 items, we need to pare it down to around half that.

I suggest keeping it to: Guide Tutorials Resources Tools Contributing Design FAQ

File: docs/hugo/content/guide/frequently-asked-questions.md (hunk 4 above)
Comment (matthchr) [reply docs/hugo/content/guide/frequently-asked-questions.md:6]: Just meant for this to be ordered in the sidebar not the menu. Have removed menu weight and left `weight`

File: docs/hugo/content/guide/installing-from-helm.md (hunk 5 above)
Comment (matthchr) [reply docs/hugo/content/guide/installing-from-helm.md:6]: Fixed, just wanted ordered in the sidebar

File: docs/hugo/content/guide/aso-controller-settings-options.md (hunk 1 above)
Comment (matthchr) [reply docs/hugo/content/guide/aso-controller-settings-options.md:75]: Fixed

File: docs/hugo/content/guide/aso-controller-settings-options.md (hunk 3 above)
Comment (matthchr) [reply docs/hugo/content/guide/aso-controller-settings-options.md:94]: We do

Comment (matthchr) [reply docs/hugo/content/guide/aso-controller-settings-options.md:94]: We do. Updated docs

**Pull request merged**